app/src/userpages/components/Avatar/CropAvatarDialog/index.jsx

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import React from 'react'
+import { I18n } from 'react-redux-i18n'
+import AvatarEditor from 'react-avatar-editor'
+
+import Dialog from '$shared/components/Dialog'
+import Slider from '$shared/components/Slider'
+
+import styles from './cropAvatarDialog.pcss'
+
+type Props = {
+    originalImage: string,
+    onClose: () => void,
+    onSave: (string) => void,
+}
+
+type State = {
+    sliderValue: number,
+}
+
+type Editor = {
+    getImage: () => HTMLCanvasElement,
+    getImageScaledToCanvas: () => HTMLCanvasElement,
+}
+
+type Ref = { current: null | Editor }
+
+class CropAvatarDialog extends React.Component<Props, State> {
+    state = {
+        sliderValue: 1,
+    }
+
+    constructor(props: Props) {
+        super(props)
+        this.editor = React.createRef()
+    }
+    editor: Ref
```

Comment:
This could be `editor: Ref = React.createRef()`. This way we can skip the constructor. 💃